cypress/support/commands.js

```diff
@@ -0,0 +1,25 @@
+// ***********************************************
```

Comment:
Why do we have this file?

Reply:
@DorianKwan Are you comfortable with deleting these?

Reply:
Duplicates have been removed!